### PR TITLE
Updated existing data converters and added new sensors

### PR DESCRIPTION
**Pull request by mravatins**
Added data converters for TEKTELIC v1, v2 and TTN v2, v3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,35 @@
-# Data Converters for TEKTELIC Network Server
-This repository contains the data converters that are to be used on TEKTELIC Network Server for TEKTELIC Sensors. These data converters can be used as a reference for other platforms (such as AWS, TTN, etc).
+# Data Converters for TEKTELIC Devices
+This repository contains the data converters that are to be used on TEKTELIC & TTN Network Server for TEKTELIC Sensors. These data converters can be used as a reference for other platforms.
```
Comment (kschluff-tektelic) [inline README.md]: I would says "& other" rather than "& TTN".

File: README.md [hunk 2]
```diff
@@ -1,33 +1,35 @@
...
 
 ### Available Sensors 
-- AC Switch and Outlet
-- Agriculture Sensor
-- Enterprise Asset Tracker (BLE Tracker)
-- GPS Asset Tracker
+- Aura Flux
```
Comment (kschluff-tektelic) [inline README.md]: "AURA / FLUX".  They are separate devices.

File: README.md [hunk 3]
```diff
@@ -1,33 +1,35 @@
...
+- Breeze
+- Clover
+- Comfort
+- eDoctor
 - Industrial Transceiver
-- Smart Room Sensor Gen3
-- Tundra Sensor
+- Kiwi
+- Orca
+- Pelican/EX
```
Comment (kschluff-tektelic) [inline README.md]: "PELICAN / PELICAN EX"

File: README.md [hunk 4]
```diff
@@ -1,33 +1,35 @@
-# Data Converters for TEKTELIC Network Server
-This repository contains the data converters that are to be used on TEKTELIC Network Server for TEKTELIC Sensors. These data converters can be used as a reference for other platforms (such as AWS, TTN, etc).
+# Data Converters for TEKTELIC Devices
+This repository contains the data converters that are to be used on TEKTELIC & TTN Network Server for TEKTELIC Sensors. These data converters can be used as a reference for other platforms.
 
 ### Available Sensors
```
Comment (kschluff-tektelic) [inline README.md:4]: As a general comment, sensor names are in uppercase.  E.g., "COMFORT", rather than "Comfort"

File: README.md (hunk 1 above)
Comment (kschluff-tektelic) [inline README.md]: Note that the Tektelic V2 data converters conform to the LoRa Alliance® LoRaWAN® Payload Codec API.

Comment (mravatins) [reply README.md]: Should I mention the version such as  LoRa Alliance® LoRaWAN® Payload Codec API 1.x.x?

**Review by kschluff-tektelic — APPROVED** [review]

**Pull request merged**